WcaOnRails/app/mailers/competitions_mailer.rb

```diff
@@ -129,6 +129,17 @@ def submit_report_nag(competition)
     )
   end
 
+  def submit_report_reminder(competition)
+    @competition = competition
+    mail(
+      from: Team.weat.email,
+      to: competition.all_delegates.pluck(:email),
+      cc: ["assistants@worldcubeassociation.org"] + delegates_to_senior_delegates_email(competition.all_delegates),
+      reply_to: competition.all_delegates.pluck(:email),
```

Comment:
Nitpick: This has `reply_to` set to all Delegates of the competition, but the body text from the Google Doc template says "please reply to this email to update and explain this to your Senior Delegate". So I think it would make sense to change this to the senior Delegate emails, similar to the existing "nag" mail.

Reply:
Fixed :wrench: 
It's funny that it was the first version. I changed it after [this](https://github.com/thewca/worldcubeassociation.org/pull/7229#discussion_r975349408) comment :smile_cat:

Reply:
Good point, but that conversation actually happened before the Board provided us with a concrete draft.